### PR TITLE
Imrpove product image in product page

### PR DESCRIPTION
**Pull request by NitinCraziino**

### CODE REVIEW

File: src/app/globals.css
Comment (gkpty) [inline src/app/globals.css:1]: i prefer using css in js or extending tailwind css in the tailwind.config.ts instead of adding classes in a globals.css file

File: package.json
Comment (gkpty) [inline package.json:1]: How is this related to this PR ?

File: .env
Comment (gkpty) [inline .env:1]: Please dont commit the .env file

File: package-lock.json
Comment (gkpty) [inline package-lock.json:1]: Im using npm in this project so please dont delete this file

File: pnpm-lock.yaml
Comment (gkpty) [inline pnpm-lock.yaml:1]: avoid commiting this file

File: src/app/globals.css
Comment (NitinCraziino) [reply src/app/globals.css:1]: Fixed

File: package.json
Comment (NitinCraziino) [reply package.json:1]: Fixed

File: .env
Comment (NitinCraziino) [reply .env:1]: Fixed

File: package-lock.json
Comment (NitinCraziino) [reply package-lock.json:1]: Fixed

File: pnpm-lock.yaml
Comment (NitinCraziino) [reply pnpm-lock.yaml:1]: Fixed
